.swiftlint.yml

```diff
@@ -1,4 +1,4 @@
-whitelist_rules:
+only_rules:
```

Comment:
It looks like you pulled in some other changes - should these be done as a separate merge into upstream fluent2-colors?

Reply:
This change is part of @harrieshin's PR into main that I cherry picked to fix the recent linting issues. I don't think it's necessary to merge it separately into fluent2-colors.

Reply:
even though our git branches history is a mess, there is a value of cherry picking one commit at a time to preserve as much history